source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -129,6 +129,7 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(os.path.dirname(BASE_DIR), 'static')
```

Comment:
No link with issue #33, please remove this diff and create a new issue if you have a problem with the static configuration.